packages/playground/src/components/account/account-register/account-register.tsx

```diff
@@ -217,7 +217,7 @@ export class AccountRegister {
       ready: "Register",
       awaitingForWallet: "Check Wallet...",
       creatingAccount: "Creating your account...",
-      deployingMultisig: "Creating state channel..."
+      deployingMultisig: "Deploying state channel contract..."
```

Comment:
There's a tradeoff to be made here.

Since this is more user facing, a user ideally shouldn't care about the technical details (i.e. that it's a deployment or that it's a contract). To them it's just a realization of the concept of a channel being created.

cc @snario @ebryn

Reply:
To a nontechnical user the confusion over what "creating state channel" means is even worse

Reply:
I think it isn't worth changing this right because CREATE2 will remove the entire process and the audience presently of the Playground demo still is developers.

Reply:
If the audience is developers, `deploying contract` is better for them as well

Reply:
OK @IIIIllllIIIIllllIIIIllllIIIIllllIIIIll does that string fit within the button width without a newline?

Reply:
`deploying contract` is shorter than `creating state channel`, but `deploying state channel contract` is longer

Reply:
```suggestion
      deployingMultisig: "Deploying contract..."
```